getting-started/setup-building.rst

```diff
@@ -419,7 +419,17 @@ For example, with **Homebrew**, install the dependencies::
 
     $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
 
-Then, for Python 3.10 and newer, run ``configure``::
+Then, for Python 3.11 and newer, run ``configure``::
```

Comment:
With all these versions, it might be better to use that Sphinx extension that has different tabs on the code block

Reply:
Yeah, I was thinking that too having looked at it. It would require adding a dep, but it could be likely useful in other places where the instructions differ by version, platform, etc.—which is quite common in the devguide.

Reply:
SGTM

Reply:
Let's keep this formatting improvement out of this PR. If we want to add the inline tabs extension, we can install that in a follow-up PR and then start to make use of it.